strings/strip.py

```diff
@@ -0,0 +1,33 @@
+def strip(user_string: str, characters: str | None = None) -> str:
+    """
+    Remove leading and trailing characters (whitespace by default) from a string.
+
+    Args:
+        user_string (str): The input string to be stripped.
+        characters (str, optional): Optional characters to be removed
+                (default is whitespace).
+
+    Returns:
+        str: The stripped string.
+
+    Examples:
+        >>> strip("   hello   ")
+        'hello'
+        >>> strip("...world...", ".")
+        'world'
+        >>> strip("123hello123", "123")
+        'hello'
+    """
+    if characters is None:
+        characters = " \t\n\r"
```

Comment:
```suggestion
def strip(user_string: str, characters: str = " \t\n\r") -> str:
    """
    Remove leading and trailing characters (whitespace by default) from a string.

    Args:
        user_string (str): The input string to be stripped.
        characters (str, optional): Optional characters to be removed
                (default is whitespace).

    Returns:
        str: The stripped string.

    Examples:
        >>> strip("   hello   ")
        'hello'
        >>> strip("...world...", ".")
        'world'
        >>> strip("123hello123", "123")
        'hello'
    """
```
Instead of allowing None, you can simply have a default string argument

Reply:
Sure.